File: program_data/github_java_sort_function/6/85.java
package sorting;


public class RadixSort {

	
	public static int[] sort(int[] arr, int n) {
		int[][] a = convert(arr, n);
		int digit = a[0].length;
		
		return null;
	}
	
	public static int[][] convert(int[] arr, int n) {
		int[][] a = new int[arr.length][(int) Math.log10(n)+1];
		for (int i = 0; i < a.length; i++) {
			int num = arr[i];
			int j = a[0].length - 1;
			while (num != 0) {
				a[i][j] = num % 10;
				num /= 10;
				j--;
			}
		}
		return a;
	}
}

File: program_data/PKU_raw/53/85.c

void main()
{
	int i,num,a[300],b[300],*p;
	scanf("%d",&num);
	for (i=0;i<=(num-1);i++)
	{
		scanf("%d",&a[i]);
	}
	printf("%d",a[0]);
	for (i=1;i<=(num-1);i++)
	{	
		for (p=a;p<&a[i];p++)
		{
			if (*p==a[i]) break;
		}
		if (p==&a[i]) printf(",%d",a[i]);
	}

}